index.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Microverse Technical Document</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+  <nav id="navbar">
+    <header><h1>JS Documentation</h1></header>

```

Comment:
- [ ] Please be more polite, professional, friendly and transparent in your comments while doing code reviews. Your comment looks like you are ordering and do have not a strong reason to ask for adding something to the document.

### Your Comment

> I think your html will more semantic if you put `<nav>` inside `<header>` part.